gef.py

```diff
@@ -8433,7 +8434,7 @@ def add_command_to_doc(self, command):
         if " " in cmd:
             # do not print subcommands in gef help
             return
-        doc = class_name.__doc__ if hasattr(class_name, "__doc__") else ""
+        doc = getattr(class_name, "__doc__", "").lstrip() if hasattr(class_name, "__doc__") else ""
```

Comment:
haha, no the point is that if we use `getattr` we don't need `hasattr`

The whole line should be

`doc = getattr(class_name, "__doc__", "").lstrip()`

No conditional needed.

Reply:
oh lol